src/EventListener/PrettyErrorScreenListener.php

```diff
@@ -130,13 +139,24 @@ private function handleException(GetResponseForExceptionEvent $event)
                 break;
 
             default:
-                $this->logException($exception);
-                $this->renderTemplate('error', 500, $event);
-                break;
+                $this->renderErrorScreenByException($event);
```

Comment:
They are part of the PR. We are now using the `renderErrorScreenByException()` method as fallback.